src/module.c

```diff
@@ -6704,6 +6736,10 @@ void moduleGILBeforeUnlock() {
      * (because it's u clear when thread safe contexts are
      * released we have to propagate here). */
     server.module_ctx_nesting--;
+    if (!server.module_ctx_nesting && server.busy_module) {
+        blockingOperationEnds();
+        server.busy_module = 0;
+    }
```

Comment:
please move to the end or beginning of the function (currently it's in the middle of lines that are logically related)